shanoir_downloader.py

```diff
@@ -338,7 +349,7 @@ def download_datasets_from_ids(args):
 	dataset_ids = Path(args.dataset_ids) if args.dataset_ids else None
 	if args.dataset_ids and not dataset_ids.exists():
 		sys.exit('Error: given file does not exist: ' + str(dataset_ids))
-	study_id = args.study_id
+	study_id = args.shanoir_study_id
```

Comment:
I guess this is a typo? The argument is named `study_id` in `shanoir_downloader.py`.